dangerfile.js

```diff
@@ -246,7 +246,9 @@ const ignoreCopyrightVerifyPath = [
   'ios/playground/bundlejs',
   'ios/sdk/WeexSDK/Resources',
   'ios/sdk/WeexSDK/Sources/Layout',
-  'ios/sdk/WeexSDK/dependency/SRWebSocket'
+  'ios/sdk/WeexSDK/dependency/SRWebSocket',
+  'weex_core/Source/include',
+  'weex_core/Source/base/third_party'
```

Comment:
Change to `'weex_core/Source/base/third_party/icu'`

Reply:
done